package.json

```diff
@@ -40,10 +40,11 @@
     "build": "npm run buildMain && npm run buildBrowser && npm run buildPreload",
     "buildWindows": "npm run build && node ./scripts/buildWindows.js",
     "buildMac": "npm run build && node ./scripts/buildMac.js",
-    "buildDebian": "npm run build && node ./scripts/buildDebian.js",
-    "buildRaspi": "npm run build && node ./scripts/buildDebian.js --raspi",
+    "buildDebian": "npm run build && node ./scripts/buildDebian.js --platform=amd64",
+    "buildRaspi": "npm run build && node ./scripts/buildDebian.js --platform=armhf",
+    "buildARM64": "npm run build && node ./scripts/buildDebian.js --platform=arm64",
```

Comment:
Maybe include "linux" in the command name somewhere?

Reply:
update buildARM64 to buildLinuxArm64